src/create-pointer-sensor.ts

```diff
@@ -25,17 +28,17 @@ export const createPointerSensor = ({ id } = { id: "pointer-sensor" }) => {
   });
 
   onCleanup(() => {
-    removeSensor({ id });
+    removeSensor(id);
   });
 
   const isActiveSensor = () => state.active.sensor === id;
 
   const initialCoordinates = { x: 0, y: 0 };
 
-  let activationDelayTimeoutId = null;
-  let activationDraggableId = null;
+  let activationDelayTimeoutId: number | null = null;
+  let activationDraggableId: string | null = null;
 
-  const attach = ({ event, draggableId }) => {
+  const attach = ({ event, draggableId }: PointerAttachParams): void => {
```

Comment:
I see. I was a bit confused with this practice since I haven't seen it before. Also - not all functions and methods followed it (from what I observed) - so I just applied my style to make implementing types easier. The example above is a leftover I forgot to adapt. For now, I'll make it in line with the following rules:
- default: *multiple params*
- values related through a single entity: *object (commonly with TS interface)*
- >3 params: *object (commonly with TS inline type)*

I've PR'ed the code as-is, but I don't want to force my style on the codebase. Hopefully, you'll adapt to the changes that suit you and revert the rest.

Reply:
Thanks. Honestly I find this part of JS one of the most frustrating and often go back and forth on it myself 😄. It's so close now with default args in function definitions, we are just missing passing named parameters. Maybe one day.

In the meantime, your rules make sense to me and I see consistency throughout this code now so let's go with that.

Reply:
Yeah, that's what I understood from the code - that the format is for clearer labeling. Indeed, named parameters would be the ideal solution here without affecting the structure and performance of the code. However, I'd say modern IDEs like WebStorm, and probably VS Code with some extension handle that pretty nicely with in-code decorations (you can see parameter names alongside values you pass to the function).